### PR TITLE
Standardize id generation across components

### PR DESCRIPTION
**Pull request by hirotomoyamada**
## Description

Standardize id generation across components.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: packages/components/progress/src/circle-progress.tsx
Comment (zonoryo17) [inline packages/components/progress/src/circle-progress.tsx:1]: badge

Is it okay to include the changes to this file in this PR as well?

**Review by kaehehehe — APPROVED** [review]

File: packages/components/progress/src/circle-progress.tsx
Comment (hirotomoyamada) [reply packages/components/progress/src/circle-progress.tsx:1]: Yes, you can include it.

**Comment by hirotomoyamada** [comment]
@kaehehehe 

Please re-revirews.

**Review by kaehehehe — APPROVED** [review]

**Review by zonoryo17 — APPROVED** [review]
LGTM

**Pull request merged**